web-animations-1/Overview.bs

```diff
@@ -731,6 +753,19 @@ A <a>document timeline</a> that is associated with a {{Document}} which is not
 an <a>active document</a> is also considered to be
 <a lt="inactive timeline">inactive</a>.
 
+The [=timeline phase|phase=] of a <a>document timeline</a> at a given moment is
+the [=timeline phase|phase=] corresponding to the <em>first</em> matching
+condition from the following:
+
+<div class="switch">
+
+:   If the <a>document timeline</a> is <a lt="inactive timeline">inactive</a>:
+::  &rarr; [=timeline inactive phase|inactive=]
```

Comment:
This also seems odd to me. I don't think there are any circumstances defined in this spec for a document timeline to be inactive so we should just say it is always active. When we find circumstances where that's not the case we can list them as exceptions (e.g. XHR docs etc.).

Reply:
Done, I moved the requirements for being inactive to the phase calculation for document timeline.